### DISCUSSION TITLE
Https web login ok, but fly login failed.

### DISCUSSION THREAD

**Discussion by onlookor** [concourse/concourse#8497] [Help & Support] [answered]
As title. 
fly login returned a message:
`could not reach the Concourse server called ci:

    Get "https://localhost:8443/api/v1/info": x509: certificate signed by unknown authority`
Yes, I'm using a pair of key/cert issued by my self signed ca for TLS.
Whatever the address is - domain name, IP, or localhost - fly login returned the same message.
In the same time, I can login in browser, with IP address or localhost.
Any suggestion would be appreciated.

**Comment by onlookor** [comment]
I see the cause of the web login ok.
I had added my self signed CA to the browser's trusted CAs list.
Does fly login have something similar to enable certs issued by self signed CA？

**Comment by onlookor** [comment, marked as answer]
Ok, fly login has an --ca-cert option for this.
Excuse me.

**Reply by ritik872000** [reply]
or else you can use -k at the end of login command

**Reply by onlookor** [reply]
Thank you, It's a convenient way.
